addons/sourcemod/scripting/shavit-mapchooser.sp

```diff
@@ -757,11 +762,15 @@ void InitiateMapVote(MapChange when)
 
 	if ((when == MapChange_MapEnd && add_extend))
 	{
+		if(g_cvMapVoteEnableReRoll.BoolValue)menu.AddItem("reroll", "Reroll Maps");
```

Comment:
code style inconsistency, does this even compile like this?

Reply:
you want me to write it like this? (also it does work this way as is)
`if ((when == MapChange_MapEnd && add_extend))
	{
		if(g_cvMapVoteEnableReRoll.BoolValue){
			menu.AddItem("reroll", "Reroll Maps");
		}
		menu.AddItem("extend", "Extend Current Map");
		gB_instantChange = false;
	}
	else if (when == MapChange_Instant)
	{
		if(g_cvMapVoteEnableReRoll.BoolValue){
			menu.AddItem("reroll", "Reroll Maps");
		}
		menu.AddItem("dontchange", "Don't Change");
		gB_instantChange = true;
	}`